composer.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "silverstripe/theme-fontpicker",
+    "description": "Customise site fonts from the admin settings",
+    "type": "silverstripe-vendormodule",
+    "keywords": [
+        "silverstripe"
+    ],
+    "license": "BSD-3-Clause",
+    "require": {
+        "silverstripe/framework": "^4.1",
+        "silverstripe/fontpicker": "*"
```

Comment:
When we have a proper release of theme, this needs to be updated. Could we set it to something like `^0.0.x-dev`